nether/portal.lua

```diff
@@ -176,6 +199,62 @@ minetest.register_chatcommand("from_hell", {
 	end
 })
 
+-- Useful for debugging Nether player state tracking. Written by Deathwing777
+minetest.register_chatcommand("in_hell", {
+	params = "[<player_name>]",
+	description = "Is the player in hell?",
+	func = function(name, pname)
+		if not minetest.check_player_privs(name, {nether=true}) then
+			return false,
+				"You need the nether priv to execute this chatcommand."
+		end
+		if not player_exists(pname) then
+			pname = name
+		end
+		local player = minetest.get_player_by_name(pname)
+		if not player then
+			return false, "Something went wrong."
+		end
+		
+		status = pname.." is in the "
+		if nether.is_player_in_nether(player) then
+			status = status.."NETHER!"
+		else
+			status = status.."OVERWORLD!"
+		end
+		
+		return true, status
+	end
+})
+
+minetest.register_chatcommand("update_hells_registry", {
+	params = "[<player_name>]",
+	description = "Update player state if they got to or from the nether in another way.",
+	func = function(name, pname)
+		if not minetest.check_player_privs(name, {nether=true}) then
+			return false,
+				"You need the nether priv to execute this chatcommand."
+		end
```

Comment:
I think a cleaner/simpler way to check privileges is using the `privs` field in the chat command definition. The chat commands which already exist before this Pull Request also don't use `privs` field for some reason.

Reply:
I copied and pasted the chat command above it as a template. This is exactly the same way that `from_hell` and `to_hell` check privs. I am not sure what you mean by `privs` field. Is this something that you are wanting me to fix prior to accepting a pull request?

Reply:
Here's an example where the `privs` field is used to check if the player has the `worldedit` privilege: https://github.com/red-001/more_chatcommands/blob/4faf1c5e5a710ad907cebc76c7ccd1419f31ae4d/init.lua#L178

Since the other chat commands also use `minetest.check_player_privs` instead of the `privs` field, for this Pull Request both ways to check the privilege are fine for me.